sources/world/README.md

```diff
@@ -1,45 +1,144 @@
--rename-layers data \
--sort this.properties.region_iso_code \
--o format=topojson prettify admin_regions_lvl2_v2.topo.json
+mapshaper -i combine-files naturalearth/10m_cultural/ne_10m_admin_1_states_provinces.shp \
+  /tmp/china_states.shp \
+  -merge-layers force \
+  -filter "this.properties.iso_3166_2 !== ''" \
+  -filter "this.properties.iso_a2 !== '-1'" \
+  -filter-fields "name,iso_3166_2,iso_a2,adm0_a3,admin" \
+  -rename-fields "region_iso_code=iso_3166_2,region_name=name,country_iso2_code=iso_a2,country_iso3_code=adm0_a3,country_name=admin" \
+  -simplify visvalingam interval=300 keep-shapes \
+  -rename-layers data \
+  -sort this.properties.region_iso_code \
+  -o format=topojson prettify ../../data/admin_regions_lvl2_v2.topo.json
 ```
 
 * Convert to GeoJSON (EMS v1)
 ```
-mapshaper -i combine-files 10m_cultural/ne_10m_admin_1_states_provinces.shp \
-china_states.shp \
--merge-layers force \
--filter "this.properties.iso_3166_2 !== ''" \
--filter-fields "name,iso_3166_2" \
--rename-fields "region_iso_code=iso_3166_2,region_name=name" \
--simplify visvalingam interval=1000 keep-shapes \
--sort this.properties.region_iso_code \
--o rfc7946 format=geojson prettify admin_regions_lvl2_v1.geo.json
+mapshaper -i combine-files ../../data/admin_regions_lvl2_v2.topo.json \
+  -simplify visvalingam interval=1000 keep-shapes \
+  -sort this.properties.region_iso_code \
+  -o rfc7946 format=geojson prettify ../../data/admin_regions_lvl2_v1.geo.json
+```
+
+* Fix for invalid geometries in the GeoJSON file
+```
+node ../../scripts/clean-geom.js -v ../../data/admin_regions_lvl2_v1.geo.json
+```
+
+## World Countries
+
+### Getting metrics from the World Bank database
+
+* Getting the total population
+```
+echo "iso,measure" > /tmp/population.csv &&\
+  curl -s "http://api.worldbank.org/v2/country/all/indicator/SP.POP.TOTL?format=json&per_page=300&date=2018" |\
+  jq -cr '.[1][] | select(.countryiso3code != "") | select(.value > 0) | .countryiso3code + "," + (.value | tostring)' |\
+  sort >> /tmp/population.csv
+```
+
+* Getting the area in square kilometers
+```
+echo "iso,measure" > /tmp/area.csv &&\
+  curl -s "http://api.worldbank.org/v2/country/all/indicator/AG.LND.TOTL.K2?format=json&per_page=300&date=2018" |\
+  jq -cr '.[1][] | select(.countryiso3code != "") | select(.value > 0) | .countryiso3code + "," + (.value | tostring)' |\
+  sort >> /tmp/area.csv
+```
+
+### Getting population and area data from wikidata
+
+After this process, there are still some records without area or population, we will complete the dataset with Wikidata information. To get the list of the missing countries we will filter using `csvsql` from the `csvkit` package over a finished dataset.
+
+* Get the country iso codes from the regions dataset
+```
+cat ../../data/admin_regions_lvl2_v1.geo.json |\
+  jq -r '.features[] | .properties.country_iso3_code + "," + .properties.country_iso2_code' |\
+  uniq | sort > /tmp/isos.csv
+```
+
```

Comment:
Out of scope for this PR, but maybe in a separate PR we should consider creating a Makefile to run the commands to generate these datasets?

Reply:
I already added a [Makefile](https://github.com/elastic/ems-file-service/pull/188/files/aaa845ee60252c7a2b0507f7e51ead82a716905d#diff-34a562d4e8165b7c9ca64cade71f884b) to this PR :sweat_smile:

Reply:
🙈 oops. I didn't see that. That's fantastic!